app/assets/stylesheets/base/reset.scss

```diff
@@ -85,6 +85,7 @@ textarea {
 
 // Drop default outline (Crayons provides custom)
 input[type='text'],
+input[type='number'],
```

Comment:
That was clearly missing 🤷‍♂️

Reply:
We could probably tweak this ruleset a bit. I imagine we want our custom outline on textareas, and all inputs except those of type hidden? We could do this instead. This would include `type="submit"` and `type="reset"` (buttons), but I imagine we want them to have that styling as well?

```css
textarea,
input:not([type='hidden']) {
    // awesome CSS
}
```

Reply:
I like it! I'll bring it in a separate PR.

Reply:
https://github.com/forem/forem/pull/16296